### PR TITLE
Add slack channel for p2p job

### PR DESCRIPTION
**Pull request by chetna14manku**
Add slack channel for p2p job

### CODE REVIEW

**Review by calebevans — APPROVED** [review]
/lgtm

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml [hunk 1]
```diff
@@ -2425,6 +2425,17 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-knative-serverless-operator-main-serverless-p2p-rosa-classic-417-rc-iib
+  reporter_config:
+    slack:
+      channel: '#forum-p2p-cspi'
```
Comment (mgencur) [inline ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml:2430]: I suppose your team will be watching the job when using this channel. All the other jobs for serverless-operator repo use `#serverless-ci`

Comment (chetna14manku) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml:2430]: @mgencur our plan is to monitor the initial couple of runs in our channel. After that we'll update this to `#serverless-ci`

Comment (mgencur) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml:2430]: /lgtm

**Pull request merged**